docs/user-guide/configuration/build-cache.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: main
+title: Build Cache
+category: User Guide
+menu: menu
+toc:
+    - title: Build Cache
+      url: "#build-cache"
+    - title: Example
+      url: "#example"
+    - title: Notes
+      url: "#notes"
+---
+# Build Cache
+Top-level setting that contains file paths from your build that you would like to cache. The cache is stored in a build's teardown bookend and restored in a build's setup bookend. You can limit access to the cache using the pipeline, event or job scope.
+
+| Scope  | Access |
+|---|---|
+| pipeline  | All builds in the same pipeline  |
+| event  | All builds in the same event  |
+| job  | All builds for the same job  |
+
+## Example
+
+```yaml
+cache:
+   event:
+       - $SD_SOURCE_DIR/node_modules
+   pipeline:
+       - ~/.gradle
+   job:
+       usejobcache: [/tmp/test]
+
+jobs:
+    setnpmcache:
+        image: node:6
+        steps:
+            - install: npm install
+        requires: [~commit, ~pr]
+    usenpmcache:
+        image: node:6
+        steps:
+            - ls: ls
+            - install: npm install
+        requires: [setnpmcache]
+    usegradlecache:
+        image: java:7
+        steps:
+            - ls: ls ~/
+            - install: git clone https://github.com/gradle-guides/gradle-site-plugin.git && cd gradle-site-plugin && ./gradlew build
+        requires: [~commit, ~pr]
+    usejobcache:
+        image: node:6
+        steps:
+            - ls-tmp: ls /tmp
+            - echo: echo hi > /tmp/testblah
+        requires: [~commit, ~pr]
+```
+
+In the above example, we cache the `node_modules` folder under the event scope in the `setnpmcache` build so that the downstream `usenpmcache` build within the same event can save time on `npm install`. Similarly, the pipeline-scoped `.gradle` cache can be access under all other builds in the pipeline to save time on `gradle install`. The `usejobcache` cache is available for use by the same `usejobcache` builds in subsequent events in the pipeline.
```

Comment:
```
The cache for `usejobcache` job is available for any subsequent builds of the same job.
```

I think the current sentence is a bit confusing to users since it mentions subsequent "events" in the "pipeline".